src/styles/custom.scss

```diff
@@ -1,27 +1,26 @@
 // Import Bootstrap
-@import '~bootstrap/scss/bootstrap';
```

Comment:
According to [Bootstrap's documentation](https://getbootstrap.com/docs/5.3/customize/sass/), the directive for importing their SASS code is:

```css
@import "../node_modules/bootstrap/scss/bootstrap";
```

This, of course, should now be `@use`.

Reply:
Both should work although for my preference, I would go with Bootstrap since this would force us to explicitly use className in react to make changes. For someone not familiar with React Bootstrap we would be looking at components with different names but they would effectively will be divs with different classes